populate_tables.sql

```diff
@@ -1,57 +1,11 @@
-COPY borough ("id", "title", "abbr")
-	FROM '../borough.csv'
-	DELIMITER ','
-	CSV HEADER;
+ \COPY borough ("id", "title", "abbr") FROM './borough.csv' DELIMITER ',' CSV HEADER;
 
+ \COPY land_use ("id", "description", "color") FROM './land_use.csv' DELIMITER ',' CSV HEADER;
 
-COPY land_use ("id", "description", "color")
-	FROM '../land_use.csv'
-	DELIMITER ','
-	CSV HEADER;
+INSERT INTO tax_lot SELECT SUBSTRING(bbl, 1, 10) as bbl, borough.id as borough_id, block, lot, address, land_use as land_use_id, ST_Transform(wkt, 4326) as wgs84, wkt as li_ft FROM source_pluto INNER JOIN borough ON source_pluto.borough=borough.abbr;
```

Comment:
oh even the `INSERT` queries had to be changed to a single line??

Reply:
Nope, fixed.